recipes/freebayes/meta.yaml

```diff
@@ -1,16 +1,15 @@
 package:
   name: freebayes
-  version: '1.1.0.46'
+  version: '1.2.0'
 
 source:
   git_url: https://github.com/ekg/freebayes.git
-  git_tag: 8d2b3a060da473e1f4f89be04edfce5cba63f1d3
+  git_tag: 40155b407a4bead708aaafcadf7904854c411275
   patches:
     - qcfail.diff
 
 build:
-  number: 3
-  skip: true  # [osx]
+  number: 4
```

Comment:
Reset the build number to `0` for a new version of the software.

Reply:
Thanks @Slugger70  - done!